lib/src/main/kotlin/com/lemonappdev/konsist/api/provider/KoHasPackageProvider.kt

```diff
@@ -4,6 +4,11 @@ package com.lemonappdev.konsist.api.provider
  * An interface representing a Kotlin declaration that provides information about whether it has a package.
  */
 interface KoHasPackageProvider : KoBaseProvider {
+    /**
+     * Whether the declaration has matching package.
```

Comment:
Whatever the package matches the directory structure (file path).

Also update desc to `KoPackageMatchingPathProvider.hasMatchingPath` to:
Whether the directory structure (file path) matches the package.